ffi/CMakeLists.txt

```diff
@@ -49,9 +52,13 @@ add_library(llvmlite SHARED assembly.cpp bitcode.cpp core.cpp initfini.cpp
 # Bug reported upstream at: https://bugs.llvm.org/show_bug.cgi?id=47003
 # BROKEN: llvm_map_components_to_libnames(llvm_libs all)
 if ($ENV{LLVMLITE_SHARED})
-    set(llvm_libs LLVM)
+    message("ENV LLVMLITE_SHARED")
+    set(llvm_libs lldCommon lldCore lldDriver lldELF lldReaderWriter lldWasm lldYAML LLVM)
 else()
-    set(llvm_libs ${LLVM_AVAILABLE_LIBS})
+    message("NOT ENV LLVMLITE_SHARED")
+    message(${LLVM_AVAILABLE_LIBS})
+    message("------------------------")
+    set(llvm_libs ${LLVM_AVAILABLE_LIBS} lldCommon lldCore lldELF lldMachO lldReaderWriter lldWasm lldYAML)
```

Comment:
As mentioned in a previous iteration, I think we need to find out if there's a more general / programmatic way of adding the correct lld libraries, or whether explicitly naming them is required.

Reply:
For the other platforms, I'm not sure how to get that info. I can keep trying to find it

Reply:
I'm not sure whether it's usable - my CMake is a bit rusty so I'd have to experiment with it a bit myself to see.

Reply:
Okay, it does not appear to be usable. I can't think of a programmatic way to do it. I don't really have enough experience with cpp build tools to even begin to figure this out.  I have put  the all the lld libs used into `build.py` so they are all in one place.